package/CHANGELOG

```diff
@@ -134,6 +135,8 @@ Changes
 Deprecations
   * analysis.hbonds.HydrogenBondAnalysis is deprecated in 1.0 (remove in 2.0)
 
+Testsuite
+  * add test for args order of base.AnalysisFromFunction (PR #2594)
```

Comment:
Tests don't typically get their own entry in the CHANGELOG. Otherwise, looks good to me!